README.md

```diff
@@ -18,6 +18,40 @@ To run from Heroku:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/yahoo/elide-spring-boot-example)
 
+## Docker and Containerize
+
+To containerize and run elide project locally
+
+1. Build Docker Image from Root of the project 
+   docker build -t elide/elide-spring-boot-example .
+2. Run docker container locally
+   docker run -p 8081:8080 elide/elide-spring-boot-example
+   https://localhost:8081/
```

Comment:
Should be http instead of https.

Reply:
updated the doc to have http://localhost:8081